From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Wed, 2 Mar 2016 17:07:42 +0100
Subject: [PATCH] ARM: i.MX: karo-tx6: disable power button

The power button is not part of the CPU module and can be triggered
wrongly on other baseboards. Disable it since we do not need it
currently.

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
---
 arch/arm/dts/imx6qdl-tx6x.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/dts/imx6qdl-tx6x.dtsi b/arch/arm/dts/imx6qdl-tx6x.dtsi
index 9fe6e6a6d7e4..9c5d676a74b2 100644
--- a/arch/arm/dts/imx6qdl-tx6x.dtsi
+++ b/arch/arm/dts/imx6qdl-tx6x.dtsi
@@ -14,6 +14,10 @@
 			device-path = &usdhc4, "partname:boot1";
 		};
 	};
+
+	gpio-keys {
+		status = "disabled";
+	};
 };
 
 &fec {
